src/api/root.rs

```diff
@@ -0,0 +1,90 @@
+use actix_web::{
+    dev::HttpServiceFactory,
+    web::{self, Data},
+    HttpResponse,
+};
+use serde::Deserialize;
+
+use crate::{
+    api::{error::ApiError, user_middleware::UserId},
+    db::{
+        model::UserQuery,
+        types::Subscription,
+        users::{find_user_by_email, get_user, root_enforce_plus, root_set_is_admin},
+        Pool,
+    },
+};
+
+#[derive(Deserialize)]
+pub struct RootQuery {
+    email: String,
+}
+
+#[derive(Deserialize)]
+pub struct RootEnforcePlusQuery {
+    pub fxa_uid: String,
+    pub enforce_plus: Option<Subscription>,
+}
+
+#[derive(Deserialize)]
+pub struct RootIsAdminQuery {
```

Comment:
NIT: Perhaps rename to RootSetAdminQuery

Reply:
then maybe even RootSetIsAdminQuery? Since the field is named is_admin